interlok-core/src/main/java/com/adaptris/core/fs/FsHelper.java

```diff
@@ -53,6 +54,8 @@ public static File toFile(String s) throws IOException, URISyntaxException {
       return createFileReference(createUrlFromString(s, true));
     } catch (IllegalArgumentException e) {
       // Catch it from createUrlFromString(), since it's probably c:/file.
+      // strip out any uri scheme
+      s = s.replaceFirst("file://", "");
```

Comment:
@telus-terchu This line is not covered by tests but seems to be important to cope with windows path.

Reply:
@sebastien-belin-adp I removed that s = s.replaceFirst() line I think its not needed.
Also, based on the updated code, the only case an IllegalArgumentException would be thrown if it was not a "file" scheme, e.g. http://localhost/./fred. While the current logic would return a valid File object, should it be supported or should it throw an exception?

Reply:
@telus-terchu I didn't really think about that but I guess it could be supported. I can imagine some places where you could use a remote http file. Not in the FsConsumer but in other place which would read something from a file.

Reply:
@sebastien-belin-adp I've left the existing logic as is. The FsHelper.createUrlFromString() will throw an IllegalArgumentException if it encounters a non "file" scheme, while the FsHelper.toFile() will catch that exception and return a new File() regardless if it is valid.